src/main/java/com/yurjinia/common/utils/JasyptUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.yurjinia.common.utils;
+
+import lombok.experimental.UtilityClass;
+import org.jasypt.encryption.pbe.StandardPBEStringEncryptor;
+
+@UtilityClass
+public class JasyptUtils {
+
+    public static String decrypt(String value) {
+        StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
```

Comment:
initialize it within Configuration class as bean.
and then use dependency injection in the respective class for calling .decrypt() method.

We don't want to store potential dependency class within Utils.